AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -217,6 +220,21 @@ - (void)fetchData
       _player = [[AVPlayer alloc] initWithPlayerItem:_currentItem];
       _player.muted = _muted;
     }
+    __weak __typeof(self) weakSelf = self;
+    _timeObserver = [_player addPeriodicTimeObserverForInterval:CMTimeMake(1, 1) queue:NULL usingBlock:^(CMTime time){
```

Comment:
But we will lose ability to control time in ASVideoNode with this rote, no? or am i missing something?  maybe it will be better to keep it in ASVideoNode for future? for example if we will implement default scrubber UI in ASVideoNode we will need periodic time observer